File: app/src/main/java/com/chinachino/mvvm/Activity/BaseActivity.java
package com.chinachino.mvvm.Activity;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.chinachino.mvvm.R;

public class BaseActivity extends AppCompatActivity {
    @Override

    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_base);
    }
}

File: app/src/main/java/com/chinachino/mvvm/activity/BaseActivity.kt
package com.chinachino.mvvm.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.chinachino.mvvm.R

 class BaseActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_base)
    }
}